### PR TITLE
Explorer popover

### PR DESCRIPTION
**Pull request by octavioamu**
##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)

##### Refers/Fixes

Refs: #2719

### CODE REVIEW

File: app/retail/templates/shared/result.html [hunk 1]
```diff
@@ -14,26 +14,51 @@
...
       <div class="title font-subheader">
         [[:title]]
       </div>
       <div class="bounty-summary col-12">
         [[if github_comments]]
-          <div class="col-xs-1">
-            <img src={% static 'v2/images/comment.svg' %}>
-            <span class="github-comment font-caption">[[:github_comments]]</span>
-          </div>
+          <!-- <div class="col-xs-1"> -->
```
Comment (mbeacom) [inline app/retail/templates/shared/result.html]: Should we drop or uncomment this?

File: app/retail/templates/shared/result.html [hunk 2]
```diff
@@ -14,26 +14,45 @@
...
-          <div class="col-xs-1">
-            <img src={% static 'v2/images/comment.svg' %}>
-            <span class="github-comment font-caption">[[:github_comments]]</span>
-          </div>
-        [[/if]]
         <div class="info font-caption">
+          <span class="status-[[:status]]">
+            <i class="fas fa-dot-circle"></i>
+            [[if status === "open"]]
+              Ready to work
```
Comment (mbeacom) [inline app/retail/templates/shared/result.html]: trans?

File: app/retail/templates/shared/result.html (hunk 1 above)
Comment (octavioamu) [reply app/retail/templates/shared/result.html]: drooooped

File: app/assets/v2/css/base.css [hunk 3]
```diff
@@ -1028,7 +1028,7 @@ input.is-invalid {
 .ui-widget.ui-widget-content {
   border: none !important;
   border-radius: 2px;
-  background: #FAFAFA !important;
+  background: #FAFAFA;
```
Comment (thelostone-mc) [inline app/assets/v2/css/base.css]: ❤️

File: app/assets/v2/css/base.css [hunk 4]
```diff
@@ -1674,3 +1674,70 @@ div.busyOverlay {
...
+  display: flex;
+  flex-wrap: wrap;
+}
+.tag.keyword {
+  display: inline-block;
+  min-width: 0;
+  font-size: 11px;
+  background-color: #E7F0FA;
+  color: #6587AE;
+}
```
Comment (thelostone-mc) [inline app/assets/v2/css/base.css]: should we move this declaration tag.css ?

File: app/assets/v2/css/base.css [hunk 5]
```diff
@@ -1674,3 +1674,70 @@ div.busyOverlay {
 .g-text-left {
   text-align: left;
 }
+
+/* Bubble */
```
Comment (thelostone-mc) [inline app/assets/v2/css/base.css]: Should we move this to `dashboard.css` ?

File: app/assets/v2/css/dashboard.css [hunk 6]
```diff
@@ -226,13 +220,41 @@ body {
...
-  }
+.hiring-alt {
+  font-size: 11px;
+  color: #4A90E2;
+  font-weight: 700;
+  letter-spacing: 1.7px;
 }
 
-.hiring {
-  content: url('/static/v2/images/hiring.svg');
```
Comment (thelostone-mc) [inline app/assets/v2/css/dashboard.css]: could we remove the svg itself if we aren't using it anywhere else ?

File: app/assets/v2/css/base.css [hunk 7]
```diff
@@ -1674,3 +1674,70 @@ div.busyOverlay {
...
+
+.tooltip-bubble__content {
+  padding: 1.4em 1.3em 1.3em;
+}
+
+.keyword-group {
+  display: flex;
+  flex-wrap: wrap;
+}
+.tag.keyword {
```
Comment (thelostone-mc) [inline app/assets/v2/css/base.css]: Could we add an empty line here ? :P

File: app/dashboard/templates/shared/bounty-popover.html [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n %}
+<div class="">
+  <div class="tooltip-bubble__content">
+    <div class="d-flex justify-content-between mb-2">
+      <span class="status-[[:status]]">
+        <i class="fas fa-dot-circle"></i>
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/bounty-popover.html]: could we change this to`far` to meet the design ?

File: app/retail/templates/shared/result.html [hunk 9]
```diff
@@ -14,26 +14,45 @@
...
       <div class="bounty-summary col-12">
-        [[if github_comments]]
-          <div class="col-xs-1">
-            <img src={% static 'v2/images/comment.svg' %}>
-            <span class="github-comment font-caption">[[:github_comments]]</span>
-          </div>
-        [[/if]]
         <div class="info font-caption">
+          <span class="status-[[:status]]">
+            <i class="fas fa-dot-circle"></i>
```
Comment (thelostone-mc) [inline app/retail/templates/shared/result.html]: `fas` -> `far`

**Comment by thelostone-mc** [comment]
@octavioamu left a few minor questions but apart from that LGTM also should we make the title bold on the bounty and tooltip like in the [design](https://user-images.githubusercontent.com/7516920/48099791-a0fcb600-e1d5-11e8-8d2d-11e34062905a.png) ? 

screen shot 2018-11-16 at 3 54 02 pm

cc @willsputra

File: app/assets/v2/css/base.css (hunk 5 above)
Comment (octavioamu) [reply app/assets/v2/css/base.css]: I think we can reuse this bubble for other places

**Pull request merged**